rosidl_runtime_py/convert.py

```diff
@@ -104,6 +104,8 @@ def message_to_csv(
     """
     Convert a ROS message to string of comma-separated values.
 
+    If the message is serialized, then a single string of bytes is returned.
```

Comment:
Using `string` and `bytes` together here is kind of confusing. I would only use the type name actually being returned.